cypress/page/shopping-cart.page.ts

```diff
@@ -2,7 +2,7 @@ class ShoppingCartPage {
   private checkoutButton: string;
 
   constructor() {
-    this.checkoutButton = ".cart_navigation span";
+    this.checkoutButton = ".cart_navigation >[title='Proceed to checkout']";
```

Comment:
if we use `.cart_navigation a[title='Proceed to checkout']` we remove the dependency of specifying the parent relationship